media/css/products/landing.scss

```diff
@@ -5,206 +5,58 @@
 $font-path: '/media/protocol/fonts';
 $image-path: '/media/protocol/img';
 
-@import '../protocol/components/custom-menu-list';
-@import '../protocol/components/fxa-form';
-@import '~@mozilla-protocol/core/protocol/css/components/logos/logo-product-firefox';
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 
-// * -------------------------------------------------------------------------- */
-// Put spacing into variables so it's easier to adjust spacing at each break point
 
-// xs
-$v-grid-xs: $layout-xs;
-$h-grid-xs: $layout-xs;
-
-// md
-$v-grid-md: $layout-md;
-$h-grid-md: $layout-lg;
-
-// lg
-$v-grid-lg: $layout-lg;
-$h-grid-lg: $layout-xl;
-
-// * -------------------------------------------------------------------------- */
-// Protocol over-rides
-
-.mzp-c-menu-list {
-    // match the bottom margin of <p>
-    margin-bottom: 1.25em;
-}
-
-.mzp-c-menu-list-title {
-    margin-bottom: 0;
+// https://github.com/mozilla/protocol/issues/893
+.mzp-c-button svg {
+    display: inline-block;
+    height: 1.2em;
+    width: 1.2em;
+    vertical-align: text-bottom;
+    @include bidi((
+        (margin-left, 0.5ch, 0),
+        (margin-right, 0, 0.5ch),
+        (transform,  scale(1, 1), scale(-1, 1)),
+    ));
```

Comment:
https://github.com/mozilla/protocol/issues/893 is now in Protocol 19.1.0 that's here since last week. Maybe the `mzp-c-button-icon-end` could help instead of this?

Reply:
(and the [open-in-new.svg](https://github.com/mozilla/protocol-assets/blob/main/icons/open-in-new.svg) icon is also in assets if you feel fancy about just styling its colour, and getting rid of the custom inline svg)

Reply:
Inline SVG is the [recommended way](https://protocol.mozilla.org/components/detail/button--with-icon) of including the icons.

I'd like to avoid as many custom styles on this page as possible. If you think those improvements should be on all buttons [please add them to Protocol ](https://github.com/mozilla/protocol/issues) 🙂

Reply:
In the long run we probably want some kind of icon library we can use as includes as needed but I'd like to get this out without blocking on solving that bigger issue.